dist/js/styleguide.js

```diff
@@ -1,5 +1,4 @@
-$(function(){
```

Comment:
Why did you remove jQuery from here? You are using jQuery inside this module. This is possible that this module will be loaded and executed faster than jQuery. Should we add check that all libraries for this module are loaded and throw error if they are not?

Reply:
I was running into issues in testing, where the code in the `$(function(){})` wasn't executing. I can look into why.